main.go

```diff
@@ -214,6 +227,7 @@ var TemplateFunctions = template.FuncMap{
 	"camel":          coerce.CamelCase,
 	"pascal":         coerce.PascalCase,
 	"upperSnake":     coerce.UpperSnakeCase,
+	"upper":          strings.ToUpper,
```

Comment:
This isn't using the coerce package implementation, which for consistency it probably shouldn't.

`lower` was intended to be `strings.ToLower`, not a case coercion, which is also why it wasn't used as a supported case.